apps/cookielifetime.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# shellcheck disable=2034,2059
+true
+# shellcheck source=lib.sh
+. <(curl -sL https://raw.githubusercontent.com/nextcloud/vm/master/lib.sh)
+
+# Check for errors + debug code and abort if something isn't right
+# 1 = ON
+# 0 = OFF
+DEBUG=0
+debug_mode
+
+# Check if root
+root_check
+
+COOKIE_LIFETIME=$(whiptail --radiolist  "Configure after what time (in seconds) after every Login every Nextcloud user gets logged out in the Browser\nSelect one with the [ARROW] Keys and the [SPACE] key and confirm by pressing [ENTER]" "$WT_HEIGHT" "$WT_WIDTH" 4 \
```

Comment:
@enoch85 would you say a --menu fits better here?

Reply:
Haven't had time to look at this yet. 

Do what you think is best, but be consequent with the other style. :+1:

Reply:
okay, then I will leave it this way.
(because in previewgenerator we have a radiolist.